diff --git a/node_modules/@types/express-session/index.d.ts b/node_modules/@types/express-session/index.d.ts
index 692b4a8..0e03673 100755
--- a/node_modules/@types/express-session/index.d.ts
+++ b/node_modules/@types/express-session/index.d.ts
@@ -24,7 +24,7 @@ declare global {
              *
              * @see SessionData
              */
-            session: session.Session & Partial<session.SessionData>;
+            session?: session.Session & Partial<session.SessionData>;

             /**
              * This request's session ID.
