diff --git a/pino.d.ts b/pino.d.ts
index 0e2cd8da35d094503aafea6577580e3b1bb43251..69723d0ed090a50d117ef9765282fd35eebca00a 100644
--- a/pino.d.ts
+++ b/pino.d.ts
@@ -24,7 +24,7 @@ declare namespace pino {
     //// Non-exported types and interfaces
 
     type TimeFn = () => string;
-    type MixinFn<CustomLevels extends string = never> = (mergeObject: object, level: number, logger:Logger<CustomLevels>) => object;
+    type MixinFn<CustomLevels extends string = string> = (mergeObject: object, level: number, logger:Logger<CustomLevels>) => object;
     type MixinMergeStrategyFn = (mergeObject: object, mixinObject: object) => object;
 
     type CustomLevelLogger<CustomLevels extends string, UseOnlyCustomLevels extends boolean = boolean> = { 
@@ -47,7 +47,7 @@ declare namespace pino {
     * A synchronous callback that will run on each creation of a new child.
     * @param child: The newly created child logger instance.
     */
-    type OnChildCallback<CustomLevels extends string = never> = (child: Logger<CustomLevels>) => void
+    type OnChildCallback<CustomLevels extends string = string> = (child: Logger<CustomLevels>) => void
 
     export interface redactOptions {
         paths: string[];
@@ -55,7 +55,7 @@ declare namespace pino {
         remove?: boolean;
     }
 
-    export interface LoggerExtras<CustomLevels extends string = never, UseOnlyCustomLevels extends boolean = boolean> extends EventEmitter {
+    export interface LoggerExtras<CustomLevels extends string = string, UseOnlyCustomLevels extends boolean = boolean> extends EventEmitter {
         /**
          * Exposes the Pino package version. Also available on the exported pino function.
          */
@@ -82,7 +82,7 @@ declare namespace pino {
          * @param options: an options object that will override child logger inherited options.
          * @returns a child logger instance.
          */
-        child<ChildCustomLevels extends string = never>(bindings: Bindings, options?: ChildLoggerOptions<ChildCustomLevels>): Logger<CustomLevels | ChildCustomLevels>;
+        child<ChildCustomLevels extends string = string>(bindings: Bindings, options?: ChildLoggerOptions<ChildCustomLevels>): Logger<CustomLevels | ChildCustomLevels>;
 
         /**
          * This can be used to modify the callback function on creation of a new child.
@@ -231,7 +231,7 @@ declare namespace pino {
     export type SerializerFn = (value: any) => any;
     export type WriteFn = (o: object) => void;
 
-    export type LevelChangeEventListener<CustomLevels extends string = never, UseOnlyCustomLevels extends boolean = boolean> = (
+    export type LevelChangeEventListener<CustomLevels extends string = string, UseOnlyCustomLevels extends boolean = boolean> = (
         lvl: LevelWithSilentOrString,
         val: number,
         prevLvl: LevelWithSilentOrString,
@@ -241,7 +241,7 @@ declare namespace pino {
 
     export type LogDescriptor = Record<string, any>;
 
-    export type Logger<CustomLevels extends string = never, UseOnlyCustomLevels extends boolean = boolean> = BaseLogger & LoggerExtras<CustomLevels> & CustomLevelLogger<CustomLevels, UseOnlyCustomLevels>;
+    export type Logger<CustomLevels extends string = string, UseOnlyCustomLevels extends boolean = boolean> = BaseLogger & LoggerExtras<CustomLevels> & CustomLevelLogger<CustomLevels, UseOnlyCustomLevels>;
 
     export type SerializedError = pinoStdSerializers.SerializedError;
     export type SerializedResponse = pinoStdSerializers.SerializedResponse;
@@ -348,7 +348,7 @@ declare namespace pino {
         <T, TMsg extends string = string>(obj: [T] extends [object] ? T & LogFnFields : T, msg?: T extends string ? never : TMsg, ...args: ParseLogFnArgs<TMsg> extends [unknown, ...unknown[]] ? ParseLogFnArgs<TMsg> : unknown[]): void;
     }
 
-    export interface LoggerOptions<CustomLevels extends string = never, UseOnlyCustomLevels extends boolean = boolean> {
+    export interface LoggerOptions<CustomLevels extends string = string, UseOnlyCustomLevels extends boolean = boolean> {
         transport?: TransportSingleOptions | TransportMultiOptions | TransportPipelineOptions
         /**
          * Avoid error causes by circular references in the object tree. Default: `true`.
@@ -708,7 +708,7 @@ declare namespace pino {
         crlf?: boolean;
     }
 
-    export interface ChildLoggerOptions<CustomLevels extends string = never> {
+    export interface ChildLoggerOptions<CustomLevels extends string = string> {
         level?: LevelOrString;
         serializers?: { [key: string]: SerializerFn };
         customLevels?: { [level in CustomLevels]: number };
@@ -863,7 +863,7 @@ declare namespace pino {
      * relative protocol is enabled. Default: process.stdout
      * @returns a new logger instance.
      */
-    function pino<CustomLevels extends string = never, UseOnlyCustomLevels extends boolean = boolean>(optionsOrStream?: LoggerOptions<CustomLevels, UseOnlyCustomLevels> | DestinationStream): Logger<CustomLevels, UseOnlyCustomLevels>;
+    function pino<CustomLevels extends string = string, UseOnlyCustomLevels extends boolean = boolean>(optionsOrStream?: LoggerOptions<CustomLevels, UseOnlyCustomLevels> | DestinationStream): Logger<CustomLevels, UseOnlyCustomLevels>;
 
     /**
      * @param [options]: an options object
@@ -871,7 +871,7 @@ declare namespace pino {
      * relative protocol is enabled. Default: process.stdout
      * @returns a new logger instance.
      */
-    function pino<CustomLevels extends string = never, UseOnlyCustomLevels extends boolean = boolean>(options: LoggerOptions<CustomLevels, UseOnlyCustomLevels>, stream?: DestinationStream | undefined): Logger<CustomLevels, UseOnlyCustomLevels>;
+    function pino<CustomLevels extends string = string, UseOnlyCustomLevels extends boolean = boolean>(options: LoggerOptions<CustomLevels, UseOnlyCustomLevels>, stream?: DestinationStream | undefined): Logger<CustomLevels, UseOnlyCustomLevels>;
 
     /**
      * Attach selected static members to the nested callable export, so that
@@ -895,7 +895,7 @@ declare namespace pino {
  * relative protocol is enabled. Default: process.stdout
  * @returns a new logger instance.
  */
-declare function pino<CustomLevels extends string = never, UseOnlyCustomLevels extends boolean = boolean>(optionsOrStream?: pino.LoggerOptions<CustomLevels, UseOnlyCustomLevels> | pino.DestinationStream): pino.Logger<CustomLevels, UseOnlyCustomLevels>;
+declare function pino<CustomLevels extends string = string, UseOnlyCustomLevels extends boolean = boolean>(optionsOrStream?: pino.LoggerOptions<CustomLevels, UseOnlyCustomLevels> | pino.DestinationStream): pino.Logger<CustomLevels, UseOnlyCustomLevels>;
 
 /**
  * @param [options]: an options object
@@ -903,6 +903,6 @@ declare function pino<CustomLevels extends string = never, UseOnlyCustomLevels e
  * relative protocol is enabled. Default: process.stdout
  * @returns a new logger instance.
  */
-declare function pino<CustomLevels extends string = never, UseOnlyCustomLevels extends boolean = boolean>(options: pino.LoggerOptions<CustomLevels, UseOnlyCustomLevels>, stream?: pino.DestinationStream | undefined): pino.Logger<CustomLevels, UseOnlyCustomLevels>;
+declare function pino<CustomLevels extends string = string, UseOnlyCustomLevels extends boolean = boolean>(options: pino.LoggerOptions<CustomLevels, UseOnlyCustomLevels>, stream?: pino.DestinationStream | undefined): pino.Logger<CustomLevels, UseOnlyCustomLevels>;
 
 export = pino;
